src/instance.rs

```diff
@@ -275,14 +35,15 @@ unsafe impl<T> Sync for Py<T> {}
 
 impl<T> Py<T> {
     /// Create new instance of T and move it under python management
-    pub fn new(py: Python, value: T) -> PyResult<Py<T>>
+    pub fn new(py: Python, value: impl IntoInitializer<T>) -> PyResult<Py<T>>
```

Comment:
It might be possible to use `TryInto<PyClassInitializer<T>, Error = PyErr>` instead of creating a whole new trait?

Reply:
We cannot implement it for `T: PyClass` because of reflective impl block https://doc.rust-lang.org/std/convert/trait.TryFrom.html#generic-implementations